packages/@atjson/source-html/src/converter/social-embeds.ts

```diff
@@ -0,0 +1,124 @@
+import Document, { Annotation } from "@atjson/document";
+import OffsetSource, { IframeEmbed } from "@atjson/offset-annotations";
+import UrlSource from "@atjson/source-url";
+
+function isInstagramOrTwitterBlockquote(annotation: Annotation<any>) {
+  if (annotation.type !== "blockquote") {
+    return false;
+  }
+
+  let classes = annotation.attributes.class;
+
+  if (!classes) {
+    return false;
+  }
+
+  let classList = classes.split(" ");
```

Comment:
This is lovely! Could this be extracted into a more general-purpose utility function? I think the logic here is more like `classes.split(/\s+/)`